src/vs/workbench/browser/parts/editor/editorActions.ts

```diff
@@ -106,6 +106,71 @@ export class SplitEditorAction extends Action {
 	}
 }
 
+export class JoinEditorsAction extends Action {
+
+	public static ID = 'workbench.action.joinEditors';
+	public static LABEL = nls.localize('joinEditors', "Join Editors");
+
+	constructor(
+		id: string,
+		label: string,
+		@IWorkbenchEditorService private editorService: IWorkbenchEditorService,
+		@IEditorGroupService private editorGroupService: IEditorGroupService
+	) {
+		super(id, label, 'join-editors-action');
```

Comment:
You do not need the CSS class here unless it would show up as icon, so I suggest to remove it.